### DISCUSSION TITLE
'virtualenv' is not recognized as an internal or external command

### DISCUSSION THREAD

**Discussion by rob2791** [Learsim/LearsimInstrumentAutomation#7] [General]
I had to run "python -m virtualenv ."

**Comment by axdra** [comment]
If ran on windows I think that is correct 👍 Example is on MacOS

**Comment by rob2791** [comment]
I know you said it’s incomplete, but I’ve got the webpage up and running
but the one thing I can’t figure out is how to execute the saved
procedures. Let me know if you’ve got any more documentation.

On Sun, Feb 21, 2021 at 5:47 PM Axel <notifications@github.com> wrote:

> If ran on windows I think that is correct 👍 Example is on MacOS
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Learsim/LearsimInstrumentAutomation/discussions/7#discussioncomment-391976>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJGKBCP3INY2FAKJ6TYO7KTTAGEPNANCNFSM4X7GBTDA>
> .
>
-- 
Thanks,
Rob Verdon

**Reply by axdra** [reply]
It executes by itself when the cockpit is loaded in FS2020

**Comment by MGBDave** [comment]
It took me some time to figure it out, but I have found it crashes every time when dragging the popout window, and I'm not sure why. Can you please let me know what versions are recommended for python and virtualenv? I'm running it on Windows 10.
